decidim-core/lib/tasks/upgrade/clean.rake

```diff
@@ -126,6 +126,19 @@ namespace :decidim do
         logger.info("===== Deleted #{invalid} invalid resources")
       end
 
+      desc "Update all blocked users notifications_sending_frequency setting"
+      task fix_blocked_user_notification: :environment do
```

Comment:
We need to add this task to the general call of `invalid_records` (line 7 in this file)

Reply:
yes, and @arpi-tremend will add also an entry in RELEASE_NOTES ... to make sure

Reply:
"bin/rails decidim:upgrade:clean:invalid_records" was removed from RELEASE_NOTES Line 24

Reply:
I explained myself wrongly. I meant to remove what you introduced and leave the releases notes as they were before, as we're adding the new task (`fix_blocked_user_notification`) inside of the metatask, that's already in the notes. I'm leaving the suggestion on what I have in mind.